AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/Migrations/20230417011220_InitialCreate.Designer.cs

```diff
@@ -11,8 +11,8 @@
 namespace AppSettingsManagerApi.Infrastructure.Migrations
 {
     [DbContext(typeof(SettingsContext))]
-    [Migration("20230319144328_Permissions")]
-    partial class Permissions
+    [Migration("20230417011220_InitialCreate")]
+    partial class InitialCreate
     {
         protected override void BuildTargetModel(ModelBuilder modelBuilder)
         {
```

Comment:
Getting rid of these Designer files and putting them in InitialCreate.cs was a good idea since it helps our code be more readable.